crypto/BRCryptoWalletManager.c

```diff
@@ -99,6 +100,7 @@ cryptoWalletManagerCreateInternal (BRCryptoCWMListener listener,
     cwm->state   = cryptoWalletManagerStateInit (CRYPTO_WALLET_MANAGER_STATE_CREATED);
     cwm->addressScheme = scheme;
     cwm->path = strdup (path);
+    cwm->peer = NULL;
```

Comment:
Presumably don't need this on the CWM given that it is in the `connect` call?

Reply:
Yes, that is true.  Although a comment in `connect()` notes that if we kept the peer we can avoid calling BRWalletManagerSetFixedPeer() every time.

The possible down side to calling SetFixedPeer is that it always does an explicit disconnect - which might be an issue?

Reply:
Well, with mode changes, we'd have to think about where we actually want to cache this (if at all).

Reply:
I removed `cwm->peer`